cmd/ai/sre_agent/sre_agent.go

```
@@ -0,0 +1,144 @@
+			if err != nil {
+				cmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	sreAgentCmd.Flags().StringVar(&pdURL, "pd-url", "", "PagerDuty incident URL (required)")
+	sreAgentCmd.Flags().BoolVar(&autoExecute, "auto-execute", false, "Fully automated mode without confirmations")
+	sreAgentCmd.Flags().StringVar(&outputDir, "output", "", "Output directory for sre-agent files (default: current directory)")
+
+	// Mark pd-url as required
+	sreAgentCmd.MarkFlagRequired("pd-url")
```

golangci-lint (2.12.2)

[error] Error return value of `sreAgentCmd.MarkFlagRequired` is not checked

(errcheck)